lib/src/dart_options.pb.dart

```diff
@@ -1,8 +1,12 @@
 ///
 //  Generated code. Do not modify.
 ///
+// @ignoreProblemForFile non_constant_identifier_names
```

Comment:
Maybe this is a separate patch, but this should probably be switched to the new syntax: `// ignore_for_file`

More: https://github.com/dart-lang/sdk/search?utf8=%E2%9C%93&q=ignore_for_file&type=